### PR TITLE
Add note for fluentd kubernetes daemonset on Windows

### PR DESCRIPTION
**Pull request by cosmo0920**
Closes https://github.com/fluent/fluentd-kubernetes-daemonset/issues/375

We don't have experiences for k8s daemonset on Windows
and IIS which is one of the important components on Windows.
So, we can only announce community user created k8s deamonset for
Windows.

Signed-off-by: Hiroshi Hatake <hatake@clear-code.com>

### CODE REVIEW

**Comment by repeatedly** [comment]
Could you update README template together?

**Comment by cosmo0920** [comment]
Oh. I've forgotten to modify README template.
I'd added note for k8s Windows daemonset.

File: README.md [hunk 1]
```diff
@@ -180,6 +180,16 @@ This is from [nekop's japanese article](https://nekop.hatenablog.com/entry/2018/
 
 zookeeper gem doesn't work on Debian 10, so kafka image doesn't include zookeeper gem.
 
+### Windows k8s daemonset doesn't support in this repository
```
Comment (elsesiy) [inline README.md:183]: Maybe rephrase this to: "Windows k8s daemonset not supported in this repository"

Comment (cosmo0920) [reply README.md:183]: Passive voice is more natural in title?

Comment (cosmo0920) [reply README.md:183]: Ah, I got it. Will do.

**Comment by repeatedly** [comment]
Thanks!
